Mage.Sets/src/mage/cards/r/RakdosFirewheeler.java

```diff
@@ -0,0 +1,48 @@
+package mage.cards.r;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.DamageTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.target.common.TargetCreatureOrPlaneswalker;
+import mage.target.common.TargetOpponent;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author jmharmon
+ */
+
+public final class RakdosFirewheeler extends CardImpl {
+
+    public RakdosFirewheeler(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{B}{B}{R}{R}");
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.ROGUE);
+
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(3);
+
+        Effect effect = new DamageTargetEffect(2);
+        effect.setText("it deals 2 damage to target opponent and 2 damage to up to one target creature or planeswalker");
+        Ability ability = new EntersBattlefieldTriggeredAbility(effect, false);
+        ability.addTarget(new TargetOpponent());
+        ability.addTarget(new TargetCreatureOrPlaneswalker());
```

Comment:
This needs to be "up to" one target creature or planeswalker, so you need to set the minimum to 0.